src/software/ai/hl/stp/action/chip_action_test.cpp

```diff
@@ -5,6 +5,23 @@
 #include "software/ai/intent/chip_intent.h"
 #include "software/ai/intent/move_intent.h"
 
+TEST(ChipTacticTest, getChipOrigin)
```

Comment:
nit: This test is testing more than its name suggests, it's testing all the getter functions. This tests should either split up, or the name should be changed accordingly.

Reply:
Changed name to getChipOriginDirectionDistanceMeters.